### ISSUE TITLE
Improve visualisation and tables in TOPPView

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/OpenMS#5717]
e.g., maybe in context of the refactoring or the ID datastructure visualization
Ask me if you need the code to produce something like this (here some protein/peptide visualization from mzTab):
![image](https://user-images.githubusercontent.com/5803621/146898712-21750250-879d-4cd3-88f3-9cc2b2cbf67e.png)

**Comment by timosachsenberg** [comment]
@cbielow could e.g. be used to create an mzTab viewer component

**Comment by jpfeuffer** [comment]
Maybe we should merge the GSOC PR at some point also? To at least have a protein table.

**Comment by timosachsenberg** [comment]
Agree. There were some comments from my side regarding putting them on doing a vertical Layout. This would have been my main concern

**Comment by jpfeuffer** [comment]
Hmm yes, but I don't like it ;) maximum would be drag'n'drop arrangement but I don't know how easy that is.
People rather have wider screens, not higher screens.

Another option would be a full-screen table view, where I can see benefits of vertical stacking (which however I guess is not at all possible, since the upper part is always reserved for some spectra).

**Comment by jpfeuffer** [comment]
By the way, I am a big fan of spark lines (e.g. to summarize quantities over multiple samples or plot densities):
https://github.com/dcdillon/dqtx

**Comment by timosachsenberg** [comment]
So drag  and drop it is? Though I don’t think that wide screen are wide enough to display both peptide and protein information
